choccy_chip/src/emulator/opcode.rs

```diff
@@ -210,7 +210,7 @@ impl Emu {
     /// - `OpCode`: The `OpCode` to execute.
     pub(crate) fn execute_opcode(&mut self, opcode: &OpCode) -> Result<(), OpCodeError> {
         match opcode {
-            OpCode::Nop => Err(OpCodeError::InvalidOpCode),
+            OpCode::Nop => Err(OpCodeError::InvalidOpCode), // TODO: should we sanitize addresses?
```

Comment:
Unless there is a "correct" implementation that can be referenced, I think we can decide this. One way we could do this is, to validate the opcodes during execution, and throw an error when facing an "invalid" opcode or one that references an address that is out of bounds. The bounds being x200 to x200 + size of rom. Another way is to check the rom before it is loaded in, but this may be less efficient in my opinion.